plugin.xml

```diff
@@ -30,7 +30,7 @@
 
     <engines>
         <engine name="cordova-android" version=">=3.6.0" /><!-- Requires CordovaPlugin.preferences -->
-        <engine name="cordova-ios" version="<6.0.0" />
+        <engine name="cordova-ios" version=">=6.0.0" />
```

Comment:
if the plugin doesn't work in cordova-ios 6, wouldn't this allow to install? I think this line should be removed

Reply:
I went ahead and reverted this one line so we can merge this in.. to keep the bumping of the version separated from the other PR.